apps/examples/testcase/le_tc/kernel/tc_sched.c

```diff
@@ -661,6 +661,9 @@ static void tc_sched_task_setcanceltype(void)
 	TC_ASSERT_EQ_CLEANUP("task_setcanceltype", ret_chk, OK, goto errout);
 	TC_ASSERT_EQ_CLEANUP("task_setcanceltype", tcb->flags & TCB_FLAG_CANCEL_DEFERRED, defferred_flag, goto errout);
 
+
+	task_testcancel();
```

Comment:
Is this needed for testing `task_setcanceltype()`?

Reply:
No, it is for covering task_testcancel() API.

Reply:
Yes, but `task_testcancel()` can be covered from tc_libc_sched.c which you made.
Isn't it?

Reply:
So every time we enable/disable the CONFIG_CANCELLATION_POINTS ,
either of these will get executed. Hence the duplication.